common/lib/xmodule/xmodule/js/karma_xmodule.conf.js

```diff
@@ -23,6 +23,7 @@ var options = {
         {pattern: 'common_static/common/js/vendor/backbone.js', included: true},
         {pattern: 'common_static/js/vendor/CodeMirror/codemirror.js', included: true},
         {pattern: 'common_static/js/vendor/draggabilly.js'},
+        {pattern: 'common_static/js/vendor/requirejs/text.js'},
```

Comment:
@clemente Should this have `included: true`?

Reply:
@UmanShahzad I already tried it, and then I have to fix what seems a bug in edx-added code: I saw `ReferenceError: define is not defined`, the problem was in [this line](https://github.com/edx/edx-platform/blob/master/common/static/js/vendor/requirejs/text.js#L393), and I'd change to `if (typeof define !== typeof undefined) {`.
But then it still cannot find require: `  ReferenceError: RequireJS is not defined`.
I'm now trying to convert the test file `video_autoadvance_spec.js` to use requirejs.

Reply:
(I added RequireJS to the _spec.js but it seems the error I mentioned happens earlier.)